src/engine/strat_engine/blockdevmgr.rs

```diff
@@ -82,7 +112,7 @@ impl BlockDevMgr {
                -> EngineResult<Vec<PathBuf>> {
         let devices = resolve_devices(paths)?;
         let bds = initialize(pool_uuid, devices, MIN_MDA_SECTORS, force)?;
-        let bdev_paths = bds.iter().map(|p| p.devnode.clone()).collect();
+        let bdev_paths = bds.iter().map(|p| p.borrow().devnode.clone()).collect();
```

Comment:
If initialize returns ```Vec<BlockDev>``` no need to borrow here, but below, need to extend with reference counted blockdevs.